src/components/Body.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styled from 'styled-components';
+import EpisodesList from './EpisodesList';
+import GeneralInfo from './GeneralInfo';
+
+const BodyDiv = styled.div`
+    border-radius: 8px;
+    border: 2px solid var(--green);
```

Comment:
random info (just so you know, no changes needed):

When you start in our codebase you may realize we have a different to way to get the variables used by us
It is something like `border:  2px solid ${props => props.theme.colors.green}`
This is mostly because we could not use css variables as they did not work with IE11.